deploy/iso/minikube-iso/configs/minikube_defconfig

```diff
@@ -1,54 +1,59 @@
 BR2_x86_64=y
 BR2_CCACHE=y
+BR2_GLOBAL_PATCH_DIR="../../deploy/patch"
 BR2_TOOLCHAIN_BUILDROOT_VENDOR="minikube"
 BR2_TOOLCHAIN_BUILDROOT_GLIBC=y
+BR2_KERNEL_HEADERS_VERSION=y
+BR2_DEFAULT_KERNEL_VERSION="4.9.64"
+BR2_PACKAGE_HOST_LINUX_HEADERS_CUSTOM_4_9=y
+BR2_BINUTILS_VERSION_2_29_X=y
+BR2_GCC_VERSION_7_X=y
 BR2_TOOLCHAIN_BUILDROOT_CXX=y
+BR2_GCC_ENABLE_LTO=y
 BR2_TARGET_GENERIC_HOSTNAME="minikube"
 BR2_TARGET_GENERIC_ISSUE="Welcome to minikube"
 BR2_TARGET_GENERIC_PASSWD_SHA512=y
-BR2_TARGET_ROOTFS_ISO9660_BOOT_MENU="$(BR2_EXTERNAL_MINIKUBE_PATH)/board/coreos/minikube/isolinux.cfg"
 BR2_INIT_SYSTEMD=y
+BR2_ROOTFS_DEVICE_TABLE="$(BR2_EXTERNAL_MINIKUBE_PATH)/board/coreos/minikube/permissions"
 BR2_SYSTEM_BIN_SH_BASH=y
 # BR2_TARGET_GENERIC_GETTY is not set
 BR2_ROOTFS_USERS_TABLES="$(BR2_EXTERNAL_MINIKUBE_PATH)/board/coreos/minikube/users"
-BR2_ROOTFS_DEVICE_TABLE="$(BR2_EXTERNAL_MINIKUBE_PATH)/board/coreos/minikube/permissions"
 BR2_ROOTFS_OVERLAY="$(BR2_EXTERNAL_MINIKUBE_PATH)/board/coreos/minikube/rootfs-overlay"
 BR2_LINUX_KERNEL=y
+BR2_LINUX_KERNEL_CUSTOM_VERSION=y
+BR2_LINUX_KERNEL_CUSTOM_VERSION_VALUE="4.9.64"
 BR2_LINUX_KERNEL_USE_CUSTOM_CONFIG=y
 BR2_LINUX_KERNEL_CUSTOM_CONFIG_FILE="$(BR2_EXTERNAL_MINIKUBE_PATH)/board/coreos/minikube/linux-4.9_defconfig"
+BR2_PACKAGE_GZIP=y
 BR2_PACKAGE_XZ=y
-BR2_PACKAGE_E2FSPROGS=y
+BR2_PACKAGE_FINDUTILS=y
```

Comment:
Looks like we dropped a few packages here:

```shell
BR2_PACKAGE_E2FSPROGS=y
BR2_PACKAGE_OPENVMTOOLS10=y
BR2_PACKAGE_OPENVMTOOLS10_PROCPS=y
BR2_TARGET_TZ_INFO=y		
BR2_TARGET_TZ_ZONELIST="default"		
BR2_TARGET_LOCALTIME="Etc/UTC"
```

are all missing but were included before.

Reply:
Everything but OPENVMTOOLS10 is in the generated `.config`.

The forked openvmtools package appears to be obsolete now, with buildroot at 10.1.0. I'll look at building with theirs.